lib/hsa/mcwamp_hsa.cpp

```diff
@@ -3848,6 +3853,8 @@ void HSAContext::ReadHccEnv()
     GET_ENV_INT(HCC_MAX_QUEUES, "Set max number of HSA queues this process will use.  accelerator_views will share the allotted queues and steal from each other as necessary");
     GET_ENV_INT(HCC_LAZY_QUEUE_CREATION, "Control lazy HSA queue creation (enabled by default).  Set to 0 to disable it.");
 
+    GET_ENV_INT(HCC_KERNARG_BUFFER_SIZE, "Maximum size of kernel arguments, in bytes. Default 4096.");
+    GET_ENV_INT(HCC_KERNARG_POOL_SIZE, "Number of kernel argument buffers to allocate each time pool is grown. Default 1024.");
```

Comment:
I'm working on changing how memory is allocated for kenarg pool and how it's managed so could we remove these knobs for now since they may become obsolete in a near future?  I'm fine with the rest of this PR.

Reply:
Done.  Started over with these two commits and omitted the env var changes.